src/pages/ProfilePage/Security/Security.jsx

```diff
@@ -22,24 +23,21 @@ const Security = () => {
     setOpenDialog(false);
   };
 
-  const triggerDeleteAccount = async () => {
+  const triggerDeleteAccount = () => {
     setOpenDialog(false);
-    try {
-      const response = await axios.delete('https://api.edukona.com/delete-user', {
-        headers: { Authorization: `Token ${localStorage.getItem('token')}` },
+    deleteUser()
+      .then(() => {
+        localStorage.clear();
+        navigate('/');
+        window.location.reload();
+      })
+      .catch((error) => {
+        console.error('Error deleting account: ', error);
+        toast.error('Error deleting account');
+        setTimeout(() => {
+          navigate('/');
+        }, 3000);
```

Comment:
I'm curious, why did you add this?

Reply:
If we encounter an error while deleting a user, redirecting immediately would mean the toast showing an error had occured is never seen. This gives a small amount of time for that to pop up.

Reply:
With navigate, I don't think that's true; the toast will show up. I was also wondering if we need to redirect them to the root to begin with(?)